server/checkout/index.js

```diff
@@ -0,0 +1,231 @@
+/* eslint-disable camelcase */
+// LIBRARIES
+const express = require('express');
+const router = express.Router();
+const bodyParser = require('body-parser');
+
+// might want to hide this key in .env file
+const stripe = require('stripe')('sk_test_51HjGjMLMMiRvpdjjl2kT4rA4xOQbDmjGNyRtTFpqSpFNwEf2u417KKyjgC0vVhvmS8JP6uYmJFtR0mSlumgBpKUZ00tarU8ly2');
```

Comment:
A: Hide this for sure, this is bad news for you - someone will scrape github and use this. (also probably revoke it on stripe)

B: Put magic strings in constants files!